packages/terra-dropdown-button/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+ChangeLog
+=========
+
+Unreleased
+-----------------
+* Initial stable release
```

Comment:
( Commenting here so the conversation can be threaded. Looking at you Github Engineers )

A couple questions regarding the functionality of the UI. 

Clicking the primary Button opens the Dropdown. But clicking the primary Button again does not close it (persistently). Should it? Or does it behave as expected, where clicking while it's open causes it to close and reopen?

![image](https://user-images.githubusercontent.com/6720991/60559041-e051b700-9d10-11e9-8ac7-0010c4bb6b2f.png)

The next question is whether we should update the cursor for the dropdown button text. I am unable to get a picture, but if you hover the text in the dropdown it will display as the default selection cursor. 
![image](https://user-images.githubusercontent.com/6720991/60559310-aaf99900-9d11-11e9-8f2f-4391ac30530b.png)

Reply:
It probably shouldn't reopen like that, looks to be a side effect of changing from `Popup` to `Hookshot`.
The mouse cursor should probably also change.

Reply:
Addressed in bd46997b4f8140247c8d530b18646f706b1033e5 and f3b73577383099a9bddb5383f6a820868611af97